src/bin/util/setup.rs

```diff
@@ -96,6 +96,9 @@ pub fn init_log(config: &TiKvConfig) -> GlobalLoggerGuard {
 
 #[allow(dead_code)]
 pub fn initial_metric(cfg: &MetricConfig, node_id: Option<u64>) {
+    util::metrics::monitor_threads("tikv")
```

Comment:
Why is it moved here?

Reply:
If we don't enable push, it will return directly